src/EnergyPlus/IOFiles.cc

```diff
@@ -464,87 +463,6 @@ void IOFiles::flushAll()
     if (err_stream) {
         err_stream->flush();
     }
-    if (json.json_stream) {
```

Comment:
I like seeing code go away, but I don't quite get why this is all going away.  It may be revealed as I keep reviewing.

Reply:
These stream files were only used to output the JSON/CBOR/MsgPack outputs from ResultsFramework. From instance,
```cpp
- FileSystem::writeFile<FileSystem::FileTypes::JSON>(jsonOutputStreams.json_TSstream_HVAC, root);
+ FileSystem::writeFile<FileSystem::FileTypes::JSON>(jsonOutputFilePaths.outputTSHvacJsonFilePath, root);
```
the `root` json object is output to `writeFile` and written to the `json_TSstream_HVAC` stream, all in a single call. The stream is not written to in any other location. We can directly open and write to the file instead of using the stream if that is the case. I wrote the following in the git commit:

> The JSON streams were there initially for unit testing, however, it is
better and easier to test on the getJSON() function, which is what the
existing unit tests are doing. Now the output files can use the faster
file writing methods.

Reply:
Thanks! :+1: